### PR TITLE
AAVE migration

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7145]

**Comment by FrederikBolding** [comment]
Has not been tested yet due to gas prices, but good for code review @blurpesec

File: src/features/AaveTokenMigration/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    fromAddress: payload.account.address,
+    spenderAddress: MIGRATION_CONTRACT,
+    chainId: DEFAULT_NETWORK_CHAINID,
+    hexGasPrice: inputGasPriceToHex(payload.gasPrice)
+  });
+
+export const createMigrationTx = (
+  payload: ITokenMigrationFormFull
+): Omit<ITxObject, 'nonce' | 'gasLimit'> => {
+  const data = AaveMigrator.migrateFromLEND.encodeInput({ amount: payload.amount });
```
Comment (blurpesec) [inline src/features/AaveTokenMigration/helpers.ts:23]: I'm assuming this is where the wrong amount came in. Here was my test approve tx: https://etherscan.io/tx/0x5ecb4ea41154b1a8f6cc3cb8af5f7162f33cd59c92708b337fc69efd67310e17
And my test migrate tx: https://etherscan.io/tx/0x4be2459a0fe228764f675b337d8228452d2a04e448cb32c166b42ba8327d180e

**Review by blurpesec — CHANGES_REQUESTED** [review]
I went through the whole flow. It approved the correct amount, but migrated the wrong amount.

After you proceed past the flow is it supposed to remove the user action from the table? Also, I think it didn't re-scan the account for token balances post-migration success.

**Review by blurpesec — APPROVED** [review]

**Comment by FrederikBolding** [comment]
Successfully tested flow: https://etherscan.io/tx/0x92c711003696a3d08289df931bd87b408186d078215649801d7189f6a944a785

**Pull request merged**